app/views/reports/instrument_unavailable_reports/_report_rows.html.haml

```diff
@@ -0,0 +1,17 @@
+%tbody
```

Comment:
Partially duplicated. This is another place that uses `@numeric_columns` to determine when to apply the `.currency` class.